### ISSUE TITLE
Support disabling Chrome Custom Tabs

### ISSUE THREAD

**Issue by yodaldevoid** [zulip/zulip-mobile#3478]
Some users (myself included) prefer not to use Chrome and would rather links never go to Chrome in the first place. As the option to disable Chrome Custom Tabs is a per-application option, the Zulip app should provide an option to not use Chrome Custom Tabs on Android.

**Comment by jainkuniya** [comment]
Yup! sometimes I too feels the requirement of this. This should a straight forward.

**Comment by borisyankov** [comment]
@jainkuniya I am removing the `bite size` tag as I don't think this is as simple to implement.

We should consider:
 * Android vs iOS - do we need different behavior or not? CustomTabs on Android vs SafariView on iOS
 * what are we doing for cases like OAuth? It should be possible to use an external browser but might be impossible to close it to get back to the app
 * where do we set the config? We would need to wire all `openLink` calls with Redux, not a huge deal, but not 'bite size'
 * we should understand the customers need first also :)

**Comment by borisyankov** [comment]
@yodaldevoid what are your motivation for needing this feature?

Do you have another, non-Chrome browser that you want to use?
Is it something against Google/Chrome? (so you really don't ever want Chrome used by the app)
Is it just that you want the link to be opened in a real (default) browser, not inside the app?

The solution will definitely vary depending on what is the problem we are trying to solve.

**Comment by yodaldevoid** [comment]
@borisyankov Looking into this further, it seems I was slightly mistaken about how Chrome Custom Tabs (CCTs) work. They function by providing an in-app browser utilizing the default browser, be that Chrome, Firefox, or what have you. Because of this, one of the possible complaints of CCTs (the fact that links are getting opened in Chrome to begin with rather than my default browser) is moot.

Personally, I use Firefox everywhere and having the page load up in a CCT slows me down when I really want it opened up in full Firefox. Honestly, I don't mind CCTs getting opened, but it does get annoying when I'm opening a series of links (say, to open a bunch of relevant issues on Github that have been linked to in a thread) where without CCTs being used the links would be sent off to my browser in the background for me to look at when I want.

What I would like to see is a toggle that disables the use of CCTs. Discord has this in a app-specific category in their settings, as does Slack under advanced.
